src/commands/Minion/tearsofguthix.ts

```diff
@@ -0,0 +1,109 @@
+import { objectEntries, Time } from 'e';
+import { CommandStore, KlasaMessage } from 'klasa';
+
+import { Emoji } from '../../lib/constants';
+import { minionNotBusy, requiresMinion } from '../../lib/minions/decorators';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import { SkillsEnum } from '../../lib/skilling/types';
+import { BotCommand } from '../../lib/structures/BotCommand';
+import { TearsOfGuthixActivityTaskOptions } from '../../lib/types/minions';
+import { formatDuration, formatSkillRequirements } from '../../lib/util';
+import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			cooldown: 3,
+			categoryFlags: ['minion'],
+			examples: ['+tearsofguthix'],
+			aliases: ['tog'],
+			description:
+				'Allows you to visit Juna and drink from the Tears of Guthix for some XP in your lowest xp skill.'
+		});
+	}
+
+	@minionNotBusy
+	@requiresMinion
+	async run(msg: KlasaMessage) {
+		const skillReqs = {
+			[SkillsEnum.Firemaking]: 49,
+			[SkillsEnum.Crafting]: 20,
+			[SkillsEnum.Mining]: 20
+		};
+		const ironmanExtraReqs = {
+			[SkillsEnum.Smithing]: 49,
+			[SkillsEnum.Thieving]: 36,
+			[SkillsEnum.Slayer]: 35
+		};
```

Comment:
Move these outside of the command, and maybe (not sure if it will work) `as const` them to prevent the need for any casting.

Reply:
Moved them but `as const` didn't help with having to use cast later.